main/src/main/scala/sbt/internal/server/BuildServerProtocol.scala

```diff
@@ -521,7 +521,8 @@ object BuildServerProtocol {
             )
           ),
           runParams.arguments,
-          defaultJvmOptions.toVector
+          defaultJvmOptions.toVector,
+          Vector.empty
```

Comment:
We should use the `envVars.value` as a default. The idea is that, if no `scalaMainClass` is provided, we run the project as it is configured in the `build.sbt`.